docs/application/tizen-studio/setup/install-sdk.md

```diff
@@ -1,40 +1,36 @@
 # Installing Tizen Studio
 
-The installer has been renewed to provide a better user experience and show the unique Tizen philosophy. Using the new installer, you can now install the basic platform as well as useful tools step by step with a few clicks.
+The installer has been renewed to provide a better user experience and show the unique Tizen philosophy. Using the new installer, you can now install the basic platform and the useful tools with a few clicks.
 
-You can use either the GUI or CLI version of the installer.
+You can use either the GUI or the CLI version of the installer.
 
 ## Using the GUI Installer
 
 To install the Tizen Studio:
 
 1. Launch the Tizen Studio installer.
 
-2. Accept the software license.
-
-   The license contains important legal notices for using the Tizen Studio. Read it fully, and click **Accept** only if you agree with the license statement.
+2. Accept the software license by reading the entire agreement, and click **Accept** only if you agree with the license statement:
 
    ![Tizen Studio License Agreement](./media/install_sdk_license.png)
 
-3. Set the SDK and data location.
-
-   The Tizen Studio and its data directory are installed in the location specified in the Installation settings step. To change the directory, click the **...** button and specify a new directory. If the new directory is valid (no errors are shown), click **Install** at the bottom of the installer.
+3. Click ![Button](./media/advanced_conf_browse.png) and specify a new directory to set the SDK and data location. If the new directory is valid and shows no errors, click **Install**:
```

Comment:
I think text is better than image to maintain docs. Please use text: **...** button.